### PR TITLE
postgres onconflict support in 9.5

### PR DESCRIPTION
**Pull request by parisholley**
https://www.postgresql.org/docs/9.5/static/sql-insert.html

allows DO NOTHING, or DO UPDATE by passing an object of key/values

### CODE REVIEW

File: src/postgres.js [hunk 1]
```diff
@@ -7,6 +7,56 @@ squel.flavours['postgres'] = function(_squel) {
   cls.DefaultQueryBuilderOptions.autoQuoteAliasNames = false;
   cls.DefaultQueryBuilderOptions.useAsForTableAliasNames = true;
 
+  // ON DUPLICATE KEY UPDATE ...
```
Comment (hiddentao) [inline src/postgres.js]: Please remove this comment.

**Comment by hiddentao** [comment]
Looks good, thanks! Just a small change required regarding a comment.

**Comment by parisholley** [comment]
@hiddentao removed

**Comment by hiddentao** [comment]
5.4.3 published

**Pull request merged**